staging/src/k8s.io/code-generator/cmd/informer-gen/generators/factory.go

```diff
@@ -257,6 +257,17 @@ func (f *sharedInformerFactory) InformerFor(obj {{.runtimeObject|raw}}, newFunc
   return informer
 }
 
+// ExistingInformers returns all existing shared informers.
+func (f *sharedInformerFactory) ExistingInformers() map[{{.schemaGroupVersionResource|raw}}]GenericInformer {
```

Comment:
Rather than add `ExistingInformers` to the factory, I personally would have added a new `SharedInformerOption` that allows the user to provide a closure like `func(informers.GenericInformer)` to run against the informer when the informer is created by the factory in `InformerFor`

Reply:
Great suggestion. I added 'WithTransform' SharedInformerOption to make this option more precise.